### ISSUE TITLE
unhandled incorrect jpeg orientation exif tag

### ISSUE THREAD

**Issue by minusf** [wagtail/Willow#104]
The `auto_orient` image operation is written fairly defensively, but we have just experienced an image poisoning attack in wagtail where the image had an existing but non-valid `orientation` value and willow failed to handle it:

```
  File "/export/apps/venv/lib/python3.10/site-packages/willow/plugins/pillow.py", line 218, in auto_orient
    if 1 <= orientation <= 8:

Exception Type: TypeError at /cms/images/chooser/
Exception Value: '<=' not supported between instances of 'int' and 'str'
```

**Comment by zerolab** [comment]
@minusf any chance you could add that image so we can have a reference for testing?

**Comment by minusf** [comment]
thank you for looking into this. i have recovered some of the files, hope it helps.  i uploaded them as a zip file: [images.zip](https://github.com/wagtail/Willow/files/9870260/images.zip)

(i think that regardless of examples out in the wild enforcing that the exif tag is `int` is a good idea)

**Comment by minusf** [comment]
this zip file has more: [images.zip](https://github.com/wagtail/Willow/files/9870288/images.zip)

**Comment by zerolab** [comment]
Relevant notes about orientation in #63

**Comment by zerolab** [comment]
Fixed in v1.6.3 / 1.7.0
